thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -122,38 +121,123 @@ body {
 
 .te-search-results {
   border-bottom: 1px app-shade(black, 1) solid;
+  color: app-shade(black, 0.7);
   padding: 24px 0;
 
   &__list {
     list-style: none;
-    // margin-bottom: 24px;
+    font-weight: normal;
+    margin-bottom: 0;
+
+    &--summary-block {
```

Comment:
`--` should be used for modifier such as `--red` `--dark` `--small` `--editable` etc. 
Please use better naming as `--summary-block` `--details-block` indicates another element.

Reply:
yes - fixed